components/global/footer/Footer.tsx

```diff
@@ -1,127 +1,56 @@
-import { Facebook, Instagram } from "lucide-react"
 import Link from "next/link"
 
-import Icon from "../../shared/icon/Icon"
+import Icon from "@/components/shared/icon/Icon"
 
 export default function Footer() {
   return (
-    <footer className="bg-background-overlay py-12">
+    <footer className="bg-background-overlay text-typo-body-md py-12 text-center lg:text-left">
       <div className="content-container">
-        <div className="grid grid-cols-1 gap-8 md:grid-cols-2 lg:grid-cols-4">
-          <div>
-            <h2 className="mb-4 text-xl font-semibold">Åbningstider</h2>
-            <p className="mb-4">
-              Bibliotekerne lorem ipsum consectetur, adipisci velit, sed quia non numquam eius modi
-              tempora incidunt ut labore.
-            </p>
-            <Link href="#" className="underline">
-              Se bibliotekernes åbningstider
-            </Link>
-          </div>
-
-          <div>
-            <h2 className="mb-4 text-xl font-semibold">Om Bibliotekerne</h2>
-            <ul className="space-y-2">
-              {[
-                "Brug af bibliotekerne",
-                "Erstatninger og gebyrer",
-                "Opret bruger",
-                "Biblioteket - A til Å",
-                "Internetadgang",
-                "Print, scan og kopi",
-                "Booking af lokaler",
-                "Nyhedsbrev",
-              ].map(item => (
-                <li key={item}>
-                  <Link href="#" className="hover:underline">
-                    {item}
-                  </Link>
-                </li>
-              ))}
-            </ul>
-          </div>
-
-          <div>
-            <h2 className="mb-4 text-xl font-semibold">Online tilbud</h2>
-            <ul className="mb-8 space-y-2">
-              {["Litteratursiden", "Bibliotek.dk", "eReolen", "Filmstriben"].map(item => (
-                <li key={item}>
-                  <Link href="#" className="hover:underline">
-                    {item}
-                  </Link>
-                </li>
-              ))}
-            </ul>
-            <h2 className="mb-4 text-xl font-semibold">Text</h2>
-            <ul className="space-y-2">
-              {[
-                "Lorem ipsum a la lorem ip",
-                "Bibliotekerne lorem",
-                "Labore ipsum",
-                "Adipisci velit le lorem",
-              ].map(item => (
-                <li key={item}>
-                  <Link href="#" className="hover:underline">
-                    {item}
-                  </Link>
-                </li>
-              ))}
+        <div className="flex flex-col flex-nowrap lg:flex-row lg:justify-between">
+          <div className="col-span-full">
+            <h2 className="text-typo-subtitle-lg col-span-full mb-4">Information</h2>
+            <p className="max-w-[600px]">GO er børnenes digitale indgang til folkebiblioteket.</p>
+            <ul className="mt-10 grid gap-y-4">
```

Comment:
grid is not necessary here. If grid was added to use `gap-y-4` then I would suggest removing grid and use `space-y-4` instead or `flex-col`

```suggestion
            <ul className="mt-10 space-y-4">
```

```suggestion
            <ul className="mt-10 flex flex-col gap-y-4">
```